docs/hello-react-navigation.md

```diff
@@ -12,32 +12,41 @@ Lets start by demonstrating the most common navigator, `createStackNavigator`.
 
 ## Creating a stack navigator
 
-`createStackNavigator` is a function that returns a React component. It takes _a route configuration object_ and, optionally, _an options object_ (we omit this below, for now). `createAppContainer` is a function that retuns a React component to take as a parameter the React component created by the `createStackNavigator`, and can be directly exported from `App.js` to be used as our App's root component.
+`createStackNavigator` is a function that returns a Navigator. Navigator is an object which contains of `Screen` and `Navigator`. Both of them are React object, but does not render anything itself. They are used for configuring behavior of navigator. `Navigator` should contains `Screen`s as children defining routes. `NavigationContainer` is a root component for navigation. This component must wrap all navigators structure.
```

Comment:
> Both of them are React object

Both of them are React components

> but does not render anything itself

The navigator does render the actual content. Though I think we can remove this sentence as it's just an implementation.